packages/block-editor/src/components/block-list/block-mobile-floating-toolbar.native.js

```diff
@@ -7,34 +7,16 @@ import { View, TouchableWithoutFeedback } from 'react-native';
  * Internal dependencies
  */
 import styles from './block-mobile-floating-toolbar.scss';
-/**
- * WordPress dependencies
- */
-import { createSlotFill } from '@wordpress/components';
-
-const { Fill, Slot } = createSlotFill( 'FloatingToolbar' );
 
 function FloatingToolbar( { children } ) {
 	return (
-		<Fill>
-			{ ( { innerFloatingToolbar } ) => {
-				return (
-					<TouchableWithoutFeedback>
-						<View
-						// Issue: `FloatingToolbar` placed above the first item in group is not touchable on Android.
-						// Temporary solution: Use `innerFloatingToolbar` to place `FloatingToolbar` over the first item in group.
-						// TODO: `{ top: innerFloatingToolbar ? 0 : -44 }` along with `innerFloatingToolbar` should be removed once issue is fixed.
-							style={ [ styles.floatingToolbarFill, { top: innerFloatingToolbar ? 0 : -44 } ] }
-						>{ children }
-						</View>
-					</TouchableWithoutFeedback>
-				);
-			} }
-
-		</Fill>
+		<TouchableWithoutFeedback>
+			<View
+				style={ styles.floatingToolbarFill }
```

Comment:
Since we aren't using a fill here, maybe we can change the name? I seem to remember this convo from a previous iteration, but can't seem to find it now, so let me know if there's a rationale for leaving this named as is.

Reply:
Sure, forgot about it. There is a [link](https://github.com/WordPress/gutenberg/pull/17769#discussion_r331875070) to which you refer.